src/crates.rs

```diff
@@ -0,0 +1,74 @@
+use crate::{
+    api,
+    commands::{Args, Result},
+};
+
+use lazy_static::lazy_static;
+use reqwest::{blocking, header};
+use serde::Deserialize;
+
+const USER_AGENT: &str = "rust-lang/discord-mods-bot";
+
+lazy_static! {
+    static ref HTTP: blocking::Client = blocking::Client::new();
```

Comment:
Is there a specific reason for making this a global static?

Reply:
reqwest's docs say to reuse clients if possible and i didn't really know where else to put it - other commands in the future could need to have a http client for whatever reason too. i guess i could pass it in to each command through Args?

Reply:
Yeah im not super pleased with making it a static global like that.  I just made a [PR](#29) adopting some of your changes regarding the Args type.  I am going to merge that in and then we can look into adding the http client as an owned type in Commands.